connectors/katalog/Makefile

```diff
@@ -13,14 +13,24 @@ generate: $(TOOLBIN)/oapi-codegen $(TOOLBIN)/crdoc $(TOOLBIN)/openapi2crd
 	$(TOOLBIN)/openapi2crd manifests/spec.yaml -g katalog.m4d.ibm.com/v1alpha1/Asset -o $(ROOT_DIR)/charts/m4d-crd/templates/katalog.m4d.ibm.com_assets.yaml
 	$(TOOLBIN)/crdoc --template ./docs/main.tmpl -c ./docs/toc.yaml -r $(ROOT_DIR)/charts/m4d-crd/templates/ -o ./docs/README.md
 
-.PHONY: build
-build: generate
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o bin/katalog main.go
-
-.PHONY: clean
-clean: docker-rmi
-	rm bin/*
+docker-build: generate
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o katalog main.go
+	docker build . -t ${IMG} -f Dockerfile
+	rm katalog
 
 .PHONY: run
 run: generate
 	go run main.go run
+
+.PHONY: test
+test:
+ifeq ($(NEED_TEST_COVERAGE_STATISTICS),1)
+	go test -v ./... -coverprofile=coverage.out
+	go tool cover -func=coverage.out
+    ifeq ($(NEED_TEST_COVERAGE_STATISTICS_IN_HTML),1)
+		go tool cover -html=coverage.out
+    endif
+	rm coverage.out
+else
+	go test -v ./...
+endif
```

Comment:
Please change to just `go test -v ./...`. When we handle coverage tooling it will be part of the entire project.